lib/ansible/module_utils/vmware.py

```diff
@@ -23,26 +24,44 @@
 from ansible.module_utils._text import to_text
 from ansible.module_utils.six import integer_types, iteritems, string_types
 from ansible.module_utils.basic import env_fallback
+from ansible.module_utils import six
 
 
 class TaskError(Exception):
     pass
 
 
-def wait_for_task(task):
+def wait_for_task(task, max_backoff=64, timeout=3600):
+    """Wait for given task using exponential back-off algorithm.
+
+    Args:
+        task: VMware task object
+        max_backoff: Maximum amount of sleep time in seconds
+        timeout: Timeout for the given task in seconds
+
+    Returns: True with result for successful task
```

Comment:
I'd also explicitly state that it returns a tuple. With current wording it seems like it returns just `True` in a certain case.

Reply:
Done.